src/components/Dashboard/MonthlyGoal/MonthlyGoal.tsx

```diff
@@ -181,6 +183,14 @@ const MonthlyGoal = ({
                             currencyFormat(goal, currencyCode, locale)
                           )}
                         </Typography>
+                        {!staffEnteredGoal && (
+                          <Button
+                            component={NextLink}
+                            href={`/accountLists/${accountListId}/settings/preferences?selectedTab=${PreferenceAccordion.MonthlyGoal}`}
+                          >
+                            {t('Set Monthly Goal')}
+                          </Button>
+                        )}
```

Comment:
This is prob for another PR, but we might want to scroll down to the opened Tab on the preferences page

Reply:
What do you mean? It already scrolls down to the monthly goal accordion, opens it, and focuses the input. Or is that not working for you?

https://github.com/user-attachments/assets/48236f46-7f00-4461-97cb-00af1ce43698

Reply:
On mine, it didn't but as long as it's working on yours